### ISSUE TITLE
Путин лучший, Леша Навальный ублюдок

### ISSUE THREAD

**Issue by itmankh** [smartvotedocs/votesmart2021#40]
Решили испортить жизнь ещё и программистам? Когда ркн начнёт блокировать гитхаб, вашего лешу на зоне отпетушат как можно сильнее, и будут проклинать до конца его дней

**Comment by aeiouaeiouaeiouaeiouaeiouaeiou** [comment]
Есть инфа от знающего человека что скоро будут реальные изменения...

**Comment by zgfif** [comment]
Типичный бот, создал акканунт лишь бы нагадить умному голосованию.

**Comment by itmankh** [comment]
> Типичный бот, создал акканунт лишь бы нагадить умному голосованию.

Нет, не бот. Просто нормальные программисты (умеющие логически мыслить), никогда не назовут это голосование "умным" и уж точно не будут голосовать за вашего подвального)))

**Comment by ax-va** [comment]
Кремлеботы набежали))

**Comment by ax-va** [comment]
> > Типичный бот, создал акканунт лишь бы нагадить умному голосованию.
> 
> Нет, не бот. Просто нормальные программисты (умеющие логически мыслить), никогда не назовут это голосование "умным" и уж точно не будут голосовать за вашего подвального)))

Только ты не программист, а кремлебот

**Comment by zgfif** [comment]
> > Типичный бот, создал акканунт лишь бы нагадить умному голосованию.
> 
> Нет, не бот. Просто нормальные программисты (умеющие логически мыслить), никогда не назовут это голосование "умным" и уж точно не будут голосовать за вашего подвального)))

Если ты не бот тогда где твои репозитории?

**Comment by itmankh** [comment]
> > > Типичный бот, создал акканунт лишь бы нагадить умному голосованию.
> > 
> > 
> > Нет, не бот. Просто нормальные программисты (умеющие логически мыслить), никогда не назовут это голосование "умным" и уж точно не будут голосовать за вашего подвального)))
> 
> Если ты не бот тогда где твои репозитории?

Ты серьезно думаешь что я стал бы говорить такое с основного аккаунта?) Серьезно?) Видимо, логически мылить ты не умеешь, я понял)

**Comment by yazver** [comment]
РКН блокирует ресурсы, но плохие не они. Чудная логика. 😏

**Comment by yazver** [comment]
> Ты серьезно думаешь что я стал бы говорить такое с основного аккаунта?) Серьезно?) Видимо, логически мылить ты не умеешь, я понял)

То есть любитель втихаря нагадить?

**Comment by skobkin** [comment]
![Lidia Arkadievna](https://kuban.ec/mediawiki/images/thumb/Arkadievna.png/400px-Arkadievna.png)

**Comment by yazver** [comment]
Не удивлюсь, если работник РКН какой-нибудь.

**Comment by darkAlert** [comment]
Пишите на него Report Abuse, аккаунт новый, сразу понятно что отдел Э

**Comment by itmankh** [comment]
> РКН блокирует ресурсы, но плохие не они. Чудная логика. 😏

Ты глубже думай) Я погляжу, стадный инстинкт работает на ура)) Вам всем втёрли про то что "леша навальный молодец, против такой политики и тд и тп", только вот про то, что он сотрудничает с пендосами и делает всё по их указанию, почему-то никто не говорит. Мда.

**Comment by itmankh** [comment]
> Пишите на него Report Abuse, аккаунт новый, сразу понятно что отдел Э

Аккаунт новый?) Посмотри заходы на гх. Он не новый, а просто не основной))))

**Comment by skobkin** [comment]
> только вот про то, что он сотрудничает с пендосами и делает всё по их указанию

"Голосование за коммунистов по указанию из США" - записал чтобы не забыть.

**Comment by uis246** [comment]
У человека првая issue на аккаунте, но он заявляет, что он программист. Seems legit.

**Comment by JoCat** [comment]
Иметь "не основной" аккаунт на гите, бля, как будто в ВК-шечку попал 🤣

**Comment by uis246** [comment]
> Видимо, логически мылить ты не умеешь

А я как погляжу ты умеешь религиозно мылить и творчески втирать
